buildSrc/src/main/groovy/local.java.container-spring-conventions.gradle

```diff
@@ -14,7 +14,11 @@ docker {
         new File(project.rootDir, 'buildSrc/docker/entrypoint-wrapper.sh'),
         new File(project.rootDir, 'buildSrc/docker/set-env-secrets.src')
 
-    buildArgs([JAR_FILE: bootJar.archiveFileName.get()])
+    def healthcheckPort = project.hasProperty('healthcheck_port') ? project.property("healthcheck_port") : 8080
+    buildArgs([
+        JAR_FILE: bootJar.archiveFileName.get(),
+        HEALTHCHECK_PORT_ARG: healthcheckPort
+    ])
```

Comment:
If a `healthcheck_port` property is set (via `gradle.properties`), then use it, otherwise use 8080 default port that spring-actuator uses.